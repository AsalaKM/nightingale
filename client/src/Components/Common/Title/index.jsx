import React, { Component } from 'react';
import { Header, Heading } from './index.style'

export default class Title extends Component {
  render() {
    const { value } = this.props;
    return (
      <Header>
        <Heading>{value}</Heading>
      </Header>
    );
  }
}