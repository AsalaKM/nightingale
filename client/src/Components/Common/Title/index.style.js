import styled from 'styled-components'

export const Header = styled.div`
    text-align:center;
`

export const Heading = styled.h1`
    border-bottom: 1.5px solid #D5D5D5;
    display: inline-block;
    padding: 3px 9px;
    font-size: 30px;
    font-weight: 100;
    margin-bottom: 5rem;
    color: #555;
`