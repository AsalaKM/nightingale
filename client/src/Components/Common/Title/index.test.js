import React from "react";
import renderer from "react-test-renderer";
import Title from "./index.jsx";

describe("Title", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Title value="Bio" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
