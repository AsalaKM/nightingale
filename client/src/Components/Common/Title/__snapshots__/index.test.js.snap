// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders correctly 1`] = `
<div
  className="sc-bdVaJa khUSez"
>
  <h1
    className="sc-bwzfXH iTXHDO"
  >
    Bio
  </h1>
</div>
`;
